File: catapult/javascript/block_check.js
const { RepositoryFactoryHttp } = require("@tech-bureau/symbol-sdk");
const { firstValueFrom } = require("rxjs");
const config = require("config");

const url = config.get("private1.url");

(async () => {
  const repo = new RepositoryFactoryHttp(url);

  const chain = await firstValueFrom(repo.createChainRepository().getChainInfo());

  console.log("height", chain.height.compact());
  console.log("finaliize height", chain.latestFinalizedBlock.height.compact());
})();

File: catapult/python/block_check.py
import httpx
from helper import get_config


def block_check():
    config = get_config()
    url = config["private1"]["url"]

    chain_info = httpx.get(url + "/chain/info").json()
    height = chain_info["height"]
    finalize_height = chain_info["latestFinalizedBlock"]["height"]
    print(f"height {height}")
    print(f"finalize height {finalize_height}")


if __name__ == "__main__":
    block_check()
